credsweeper/common/morpheme_checklist.txt

```diff
@@ -0,0 +1,1057 @@
+abort
```

Comment:
Could you explain how this list created?
I want to know about the conditions and why.

Reply:
The set was collected from false-positive credentials which matched regex plain Base64 pattern.
Then long variable name was splitted to the morphemes and the list is appended.
UnitTest  def test_morpheme_set_n(self):  verifies and optimizes the set. The changed set might be commited.